### ISSUE TITLE
No internal link creation on subtask to task conversion if language is differing from english

### ISSUE THREAD

**Issue by DeDoubleYou** [kanboard/kanboard#4965]
### Checklist

- [ x] I verified that Kanboard is correctly installed
- [ x] I verified that the problem does not come from a plugin
- [ x] I verified that the problem is not already reported
- [ x] I understand that Kanboard is in maintenance mode. It doesn't mean it's abandoned, but there is no significant feature development

### Actual behaviour

If you are using kanboard with language setting differing from english (example: de_DE) there will be no `is a parent of` <=> `is a child of` links between task and task created from subtasks generated, i. e. you can not see any internal link between the main task and the newly created taks (from subtask).

### Expected behaviour

Internal link between task and subtask (as result of conversion to task) has to be created undpendendly of the users language setting and application language settings.

### Steps to reproduce

* set the language to `Deutsch` in application settings.
* create a test project
  * create a test task
    * create a subtask within test task
    * convert subtask to task
    * check internal links section
    * you will find no internal link created

### Configuration

- Kanboard version: 1.2.22
- Database type and version: sqlite and postgres14 checked
- PHP version: 8
- OS: provided docker container
- Browser: chrome, firefox

**Comment by ianme** [comment]
Thank you for creating this issue and for giving a temporary solution!

The problem appeared after updating from 1.2.20 to 1.2.22, we had (!) KB in french and we switched to english for the moment.

**Comment by fmv1992** [comment]
I'm also affected by this. I'm using Arabic/Farsi characters in the tasks titles and no suggestions on the `(*)` show up as I type (even if I add latin characters to my task title).

**Comment by fmv1992** [comment]
I have inspected a few things and the `autocomplete` function seems to be wrong (just an idea).

**Comment by fmv1992** [comment]
Sorry for the noise, actually I have the following on my `config.php`:

```
///////////////////////////////////////////////////////////////////////////////
// Original file //////////////////////////////////////////////////////////////
///////////////////////////////////////////////////////////////////////////////
// <?php
//
// defined('ENABLE_URL_REWRITE') or define('ENABLE_URL_REWRITE', true);
// defined('LOG_DRIVER') or define('LOG_DRIVER', 'system');
///////////////////////////////////////////////////////////////////////////////

defined('ENABLE_URL_REWRITE') or define('ENABLE_URL_REWRITE', true);
defined('LOG_DRIVER') or define('LOG_DRIVER', 'stderr');
define('DEBUG', true);
```

That prevents the pop ups from showing... And by the way that pollutes my application:

* * *
![image](https://user-images.githubusercontent.com/15706854/164537619-fd65d134-dc20-46e9-9b39-0d5f4029371d.png)


* * *


* * *

Thanks for making this open source.

**Comment by Raemsna** [comment]
+1
I can reproduced the error on  v1.2.21 german language. 
Switching to english language there, the internal task is created as it should be

**Comment by tomeli5n** [comment]
Can reproduce on v1.2.21 spanish.
There are any other workaround than switching to English?
